apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -196,7 +196,7 @@ export default function ListBox({
     if (typeof setCount === 'function' && layout) {
       setCount(layout.qListObject.qSize.qcy);
     }
-  }, [layout]);
+  }, [layout, layout && layout.qListObject.qSize.qcy]);
```

Comment:
This diff is confusing me a little... this is actually what it looks like on master.. maybe something is messed up with this branch or the PR. It's probably fine but a little confusing that we see this diff.